pandas/tests/groupby/test_any_all.py

```diff
@@ -83,6 +83,23 @@ def test_any_non_keyword_deprecation():
     tm.assert_equal(result, expected)
 
 
+def test_any_apply_keyword_non_zero_axis_regression():
+    # https://github.com/pandas-dev/pandas/issues/48656
+    df = DataFrame({"A": [1, 2], "B": [0, 2], "C": [0, 0]})
+    expected = df.any(axis=1)
```

Comment:
can we construct `expected` explicitly? like
```
expected = pd.DataFrame(...)
```
(just in case there's a bug in `any` and this changes)